cfme/tests/cloud/test_tenant_quota.py

```diff
@@ -16,6 +16,15 @@
 ]
 
 
+@pytest.fixture
+def set_default(request):
```

Comment:
Can you explain why this has to be an indirect fixture? Its simply a boolean value that you can directly parametrize on

Reply:
Hi @mshriver 
- Now this fixture does not return only ```boolean``` value. So I have modified this fixture and it returns three values:  ```provisioning_entry_point```, ```reconfigure_entry_point```, ```retirement_entry_point```.
- Out of these three values ```provisioning_entry_point``` is required for this test. Hence I put  other two values to be ```None```.
- And the tests which does not require these values, this fixture returns ```None``` for those test.